src/modules/submitScore.js

```diff
@@ -1,27 +1,23 @@
-const API_URL =  "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores";
+const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores';
+const userName = document.getElementById('userName');
+const userScore = document.getElementById('userScore');
 
-function postMessage(data) {
+class Data {
+  'user':"userName.value"
+
+  'score': "userName.value"
+}
+
+function postMessage() {
   const config = {
     method: 'POST',
     headers: {
-      'Accept': 'application/json',
-      'Content-Type': 'application/json'
+      'Access-Control-Allow-Origin': 'origin',
+      'Content-Type': 'application/json',
     },
-    body: JSON.stringify(data)
-  }
-  fetch(API_URL, config)
-  .then((r) => r.json())
-  .then((data) => {
-    console.log(data)
-  })
-  const data = {
-    user: userName.value,
-    score: userScore.value
-  }
-  postMessage(data)
-  userName.value = ''
-  userScore.value = ''
-  window.location.reload()
+  };
+  fetch(API_URL, config);
+  userName.value = '';
+  userScore.value = '';
 }
-// end of submitScore function
-const setScore = document.getElementById('submit').addEventListener('click', postMessage)
\ No newline at end of file
+postMessage(Data);
```

Comment:
- [ ] When you initialized the function `postMessage`, you didn't pass in any parameters, if you call the function with parameters, it won't work. You can decide to add parameters and use the parameters in the body of the function if you need a parameter for the function.

- [ ] I can see that you want to use this function in the _**index.js**_ file. You would need to export this function then import it at the beginning of the `.js` file you want to use to call the function.

- `export postMessage`.
- Then in the _**index.js**_ file, you will do this `import postMessage from file_path`